base/config/schema.mustache

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+const deref = require('json-schema-deref-sync');
+const appSchema = require('./schemas');
+
+const schemaIndexes = {}; // TODO move this into individual src/models maybe?
+
+Object.assign(module.exports, transformSchema(schemaIndexes, appSchema));
+
+function transformSchema(indexes, schema) {
+  return deref(_.mapValues(schema, (model) => {
+    const modelIndexes = indexes[model.camelCasePlural];
+    model.properties = indexModel(modelIndexes, model.properties);
+    return model;
+  }));
+}
+
+// Note: This only adds the .indexed property to the schema's top level
+// properties and not any nested properties that it might have
+function indexModel(indexes, properties) {
```

Comment:
So now that the tables folder is gone, this is the way we set indexes on properties?

Previously this was just a way of marking fields so that the query builder could take appropriate action. Using the indexed property as a mechanism of actually creating indexes seems like an abuse of the schema.

I think we'd be better off asking people to directly set `.ensureIndex` in the model.